### ISSUE TITLE
Needs documentation

### ISSUE THREAD

**Issue by graham-sportsmgmt** [ember-cli/ember-router-generator#35]
I'd like to use this in my custom blueprint but there's no documentation.

**Comment by acorncom** [comment]
I think your best documentation for this is going to be to look at the usage shown in the tests ... https://github.com/ember-cli/ember-router-generator/blob/master/tests/generator-test.js

**Comment by graham-sportsmgmt** [comment]
Yes, I've been looking at the tests and other samples in ember-cli.  This is a request for actual documentation.
